src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -182,9 +217,42 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
         );
     };
 
+    const handlePopupConfirmation = () => {
+        // This should reset all the forms' contents
+        setOpenConfirmationPopup(false);
+        setContingencyListType(chosenContingencyListType);
+        setCreateContingencyListErr('');
+        setUnsavedChanges(false);
+        currentCriteriaBasedFilter.current = null;
+        setIsCriteriaBasedEquipmentTypeSelected(false);
+        setCurrentScript(null);
+    };
+
+    const onScriptChangeHandler = (newScript) => {
+        setCurrentScript(newScript);
+        if (newScript !== currentScript) {
```

Comment:
maybe we should compare string with `localCompare` instead of those operators...